apps/web-giddh/src/app/subscription/view-subscription/view-subscription.component.html

```diff
@@ -119,6 +119,39 @@ <h4 class="font-16">{{ commonLocaleData?.app_view }} {{ localeData?.tab_heading
                             </span>
                         </div>
                         <h4 class="border-bottom pb-3 font-16 mr-t2">{{ commonLocaleData?.app_total }}</h4>
+                        <div class="d-flex justify-content-between pt-3">
+                            <div>
+                                <p class="light-gray mb-1 font-12">Subscription Id</p>
+                                <span class="font-20">
+                                    {{ subscriptionId }}
+                                </span>
+                            </div>
+                            <div *ngIf="viewSubscriptionData?.taxTotal">
+                                <p class="light-gray mb-1 font-12">GST Tax Total</p>
+                                <span class="font-20">
+                                    <amount-field
+                                        class="d-inline-block"
+                                        [useGiddhCurrencyPipe]="false"
+                                        [amount]="viewSubscriptionData?.taxTotal"
+                                        [currencySymbol]="false"
+                                        [currencyCode]="false"
+                                    >
+                                    </amount-field>
+                                </span>
+                            </div>
+                            <div class="view-right d-flex align-items-end" *ngIf="viewSubscriptionData?.promoCode">
+                                <p class="light-gray mb-1 font-12">Promocode</p>
+                                <span class="font-20">
+                                    {{ viewSubscriptionData?.promoCode?.code }}
+                                </span>
+                            </div>
+                            <div *ngIf="viewSubscriptionData?.discount">
+                                <p class="light-gray mb-1 font-12">Discount</p>
```

Comment:
Translate whole file

Reply:
Approved with @ravindersinghdang sir that will deploy this without translation ???